### DISCUSSION TITLE
Docker container?

### DISCUSSION THREAD

**Discussion by sdondley** [tbabej/taskwiki#387] [Q&A]
I'm curious about the Docker container in this project. What's the idea behind it?

I've only dabbled a little with Docker but in the back of my mind I've been wondering about setting up a container with settings that have more advanced features and settings working out of the box. The container would have taskwiki/taskwarrior/nvim/vimwiki/taskopen and other utilities and settings for an appliance that works well out of the box.

I don't want to reinvent the wheel so I'd like to learn more about what's already here in this project and what its goals are.

**Comment by tbabej** [comment]
The docker containers are really only used to create a consistent environment in which the tests are being run :slightly_smiling_face: so the only purpose they serve is to ensure the stability of the plugin itself throughout the development process.

**Comment by sdondley** [comment]
Ok, thanks. What's your opinion on the usefulness of creating a container as I described? Do you think that's a worthwhile effort? I  may just do it anyway for the experience but curious to know what a more knowledgeable person thinks.
